src/containers/Signup/Signup.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+class Signup extends React.Component {
+    constructor (props) {
+        super(props)
+
+        this.state = {
+            inputs: {
+                username: "", 
+                email: "",
+                password: ""
+            }
+        }
+    }
+
+    handleInputChange = e => {
+        const { inputs } = {...this.state};
+        inputs[e.target.name]= e.target.value.trim();
+
+        this.setState({ inputs })
+    }
+
+    render () {
+        const { inputs } = this.state;
+        const inputArray = Object.entries(inputs);
+
+        return (
+            <>
```

Comment:
Just nitpicking here: you don't need the `<>` and `</>` if there's only one children. These "Fragments" are used if you want to have multiple nodes next to each other without a parent:

```js
render() {
    return <>
        <span>one</span>
        <span>two</span>
    </>;
}
```